### PR TITLE
jshint housekeeping

### PR DESCRIPTION
**Pull request by woutercommandeur**
This fixes most of the complaints jshint gave. It also fixes some weird issue in SpriteMaskFilter where the fs module could only be required inside the constructor for browserify to work.

### CODE REVIEW

File: src/core/renderers/webgl/filters/SpriteMaskFilter.js [hunk 1]
```diff
@@ -13,6 +12,7 @@ var AbstractFilter = require('./AbstractFilter'),
 function SpriteMaskFilter(sprite)
 {
     var maskMatrix = new math.Matrix();
+    var fs = require('fs');
```
Comment (englercj) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: Why was this moved out of the top requires? Should be outside the scope of this fn.

Comment (woutercommandeur) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: Because browserify failed and all other filters do it the same way

Comment (englercj) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: Not sure why browserify would fail, and others was me writing an example of how to do it and it got copy-pasted everywhere. fs should be required at the top of the file then used later on, not run in each constructor.

If there really is a browserify error, I'll fix it.

Comment (woutercommandeur) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: Try the current dev. It won't build. It does build when moving it into the constructor for some reason.

Comment (woutercommandeur) [inline src/core/renderers/webgl/filters/SpriteMaskFilter.js]: Want me to move it back for this pull request?

**Comment by woutercommandeur** [comment]
moved it back to the top where it should be.
